WPL/AI-usage.md

```diff
@@ -0,0 +1,693 @@
+
+Aunque no sea recomendable basar todo el contenido de un informe en la salida de una inteligencia artificial, sí es cierto que pueden ser una guía inicial en un problema, sugiriendo posibles puntos de inicio en una temática o refiriéndose a una página u escrito que tenga información más detallada o recursos interesantes. 
+
+Ocasionalmente también pueden mejorar la calidad de la redacción, al ofrecer sinónimos para ciertas palabras, formas alternativas de mostrar una misma información o más simplemente buscando errores ortográficos o semánticos. 
+
+Tras comenzar la fase de desarrollo del proyecto, se han usado inteligencias artificiales para facilitar el comienzo de las distintas secciones del trabajo. Esto ha ayudado en gran medida a tener una base estable para el proyecto. 
+
+En general, el uso promedio de los sistemas de información debe estar orientado a un apoyo o soporte básico, bien desde un punto de vista más general (estructura o información a estudiar, por ejemplo) o en un ámbito más concreto (recomendaciones a la hora de redactar, posiblemente).
+
+
+
+
+# 3. Peligros
+
+Aunque es indiscutible que las inteligencias artificiales pueden ayudar en el desarrollo de un documento y programar, no cabe lugar a duda que estos sistemas no son fiables. Al no tener capacidad propia de entendimiento o comprensión de su salida, siempre es posible que la información o código proporcionada no sea correcta o coherente. 
+
+Una base de datos pequeña, con un bias o incorrecta pueden generar respuestas que aunque estén bien formadas y parezcan razonables causen que la respuesta esté incompleta, incomprensible o directamente mal redactada. Aunque estas salidas son relativamente fáciles de detectar, siempre existe la posibilidad de que la respuesta de un modelo esté bien presentada pero basada en información errónea, que, suponiendo que el usuario ha preguntado al sistema porque no es experto en la materia, es más difícil de detectar. 
+
+Es por eso que el uso de estas tecnologías debe hacerse cautelosamente, no dando nada por sentado y contrastando la información obtenida con expertos en el tema o con conocimiento previamente verificado. 
+
+
+
+# 4. Herramientas usadas
+
+Hoy en día hay un gran repertorio de modelos de aprendizaje de lenguaje, todos con sus ventajas e inconvenientes. En este proyecto, se ha usado Chat GPT 3.5 mayormente, ya que múltiples miembros del equipo han tenido experiencias positivas con este sistema. Las interacciones que ha tenido el grupo con la inteligencia artificial han sido grabadas en la sección 5. 
+
+Otras IAs que se han usado son Builder.io y Anima, plugins de Figma que permiten traducir los “mock-ups” que se crearon para esbozar la vista de front-end a código usable en React y CSS, la base elegida para el apartado visual de la aplicación. Estos plugins no requieren prompts, si no un archivo de Figma.
```

Comment:
In Spanish, the plural form of initials doesn't change ("Una IA" -> "Varias IA").